packages/cli-lib/lang/en.lyaml

```diff
@@ -38,55 +52,27 @@ en:
         describe: "Commands for working with the config file."
         subcommands:
           set:
-            describe: "Commands for working with the config file."
-            subcommands:
+            describe: "Commands for modifying the CLI configuration"
+            promptMessage: "Select a config option to update"
+            examples:
+              default: "This is an example of something"
```

Comment:
I'm a little confused by "This is an example of something." What's the use case here?

Reply:
LOL, I had this in as a placeholder and forgot to go back and update it. It should say something along the lines of "Opens a prompt to edit config items".